launch/src/main/java/meteordevelopment/meteorclient/Main.java

```diff
@@ -6,36 +6,37 @@
 package meteordevelopment.meteorclient;
 
 import javax.swing.*;
-import java.io.File;
-import java.io.IOException;
+import java.io.*;
 import java.net.MalformedURLException;
 import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 import java.util.Locale;
+import java.util.concurrent.atomic.AtomicReference;
 
 public class Main {
-    public static void main(String[] args) throws UnsupportedLookAndFeelException, ClassNotFoundException, InstantiationException, IllegalAccessException {
-        UIManager.setLookAndFeel(UIManager.getSystemLookAndFeelClassName());
```

Comment:
why did you remove this?

Reply:
Oh shit I removed too much

Reply:
Ok should be fixed

Reply:
I didn't see that my commit was rejected, let me fix